Dockerfile

```diff
@@ -1,15 +1,32 @@
-FROM node:10-jessie
+FROM --platform=${TARGETPLATFORM:-linux/amd64} node:alpine

```

Comment:
I opened a new PR #33 based on this one with more optimizations. I think we should merge the other PR first and rebase this one.

Reply:
SGTM :)